src/crypto/store/base.js

```diff
@@ -9,3 +9,23 @@
  *
  * @interface CryptoStore
  */
+
+/**
+ * Represents an outgoing room key request
+ *
+ * @typedef {Object} OutgoingRoomKeyRequest
+ *
+ * @property {string} requestId    unique id for this request
```

Comment:
This goes over the wire as `request_id` - so either link to the docs for the wire protocol or state the protocol here please. There's no other way for me to know what is valid and what is not otherwise.